tools/pylib/boutdata/data.py

```diff
@@ -247,6 +249,50 @@ def __init__(self, filename, name="root"):
 
                         section[line[:eqpos].strip()] = value
 
+    def write_new(self, filename=None, force_overwrite=False):
```

Comment:
Could just be `write` rather than `write_new`, as it doesn't necessarily make a new file

Also `force_overwrite` could just be `overwrite`